sdk/core/Azure.Core/src/Pipeline/Policies/RetryPolicy.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Azure.Core.Pipeline.Policies
 {
-    public class RetryPolicy : HttpPipelinePolicy
+    internal class RetryPolicy : HttpPipelinePolicy
```

Comment:
At some point we may add a Storage specific retry policy that switches between geo-redundant data centers for reads and that may involve revisiting this decision, but I think it's fine for now.